hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx

```diff
@@ -44,7 +44,7 @@ class AutoReloadPanel extends React.Component<IAutoReloadPanelProps> {
         <Tooltip
           placement='bottom' title={moment(lastUpdated).format('lll')}
```

Comment:
Can we also update the tooltip to include seconds?
```suggestion
          placement='bottom' title={moment(lastUpdated).format('ll LTS')}
```